### PR TITLE
tests: add tls alpn test

### PR DESCRIPTION
**Pull request by victorjulien**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:

### CODE REVIEW

**Review by catenacyber — APPROVED** [review]
These SV tests look good to me for the feature

File: tests/tls-alpn-log-detect-02/test.rules [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alert tls any any -> any any (tls.sni; content:"icloud"; tls.alpn; content:"http/1.1"; sid:1;)
+alert tls any any -> any any (tls.subjectaltname; content:"p142-contacts.icloud.com"; tls.alpn; content:"http/1.1"; sid:2;)
```
Comment (catenacyber) [inline tests/tls-alpn-log-detect-02/test.rules:2]: Question : why 2 tests ? We test logging here as well...

File: tests/tls-alpn-log-detect-02/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+PCAP
+====
+
+Pcap recorded by Victor Julien
```
Comment (catenacyber) [inline tests/tls-alpn-log-detect-02/README.md:4]: Nit: redmine ticket reference

**Comment by victorjulien** [comment]
replaced by #1924

**Pull request closed without merging**